roles/galaxy.movedata/tasks/main.yml

```diff
@@ -19,24 +19,8 @@
 - include: export.yml
   when: export_dir_exists.stat.exists and not export_exists.stat.exists
 
-- name: Adjust proxy prefix in galaxy.ini
```

Comment:
How is this being handled now ?
We originally added this for running Galaxy using a prefix (e.g localhost:/galaxy)

Reply:
Yeah I did it and I am not sure it was the best way: the idea here is that tuning the nginx prefix and ftp host ip in a "movedata" role is pretty confusing. I am thinking of either to get it handled by the ansible-galaxy-extra role (which is not apparently the case but not clear), or by a GKS role that would be more explicit (copy additional files existing role or something appropriately renamed). Work in progress...

Reply:
But you have to ensure prefix and IP address are correct when restoring a persisted session (e.g after moving to a new cloud zone), so I think this is tightly couple to the role. I'm not sure anything can be gained by splitting this here.

Reply:
update: actually, both nginx prefix and ftp host ip can be  configured when the ansible-galaxy role plays from the [template](https://github.com/galaxyproject/ansible-galaxy/blob/master/templates/galaxy.ini.j2) updated by @natefoo 6 months ago.

we have just to set properly the galaxy_config["app:main"][ftp_upload_site]  and the galaxy_config["filter:proxy-prefix"][prefix] variable in the group_vars files.

This is not really splitting, this is moving steps to their logical location (IMHO, ansible-galaxy). If the variables are set in the GKS group_vars I don't see the issue with regard to persisted sessions. The current state for ftps is pretty "hacky" ( a curl icanhazip followed by a sed replacement !), nothing to change for the nginx prefix.

Reply:
Sure, as I said it would be easier to judge if that part was also implemented in this PR.

Reply:
sure. WP !